graph/src/components/ethereum/types.rs

```diff
@@ -11,6 +11,8 @@ use crate::{
     prelude::{BlockNumber, DeploymentHash, EntityKey, ToEntityKey},
 };
 
+const BYZANTIUM_FORK_BLOCK_NUMBER: u64 = 4_370_000;
```

Comment:
You can't do this, the block number will be different for different networks such as testnets. Simply assume that if all gas was used, that is a failed transaction regardless of the block.